src/main/java/org/prebid/server/bidder/conversant/ConversantBidder.java

```diff
@@ -93,22 +94,20 @@ public ConversantBidder(String endpointUrl, JacksonMapper mapper) {
     }
 
     private BidRequest createBidRequest(BidRequest bidRequest, List<BidderError> errors) {
-        final App app = bidRequest.getApp();
-        if (app != null && StringUtils.isBlank(app.getId())) {
-            throw new PreBidException("Missing app id");
-        }
-
         final List<Imp> modifiedImps = new ArrayList<>();
         Integer extMobile = null;
         String extSiteId = null;
         final Site site = bidRequest.getSite();
```

Comment:
This and `app` variable can be declared right before `validateSiteAppId(...)` method, since they are used further.